From cb821d150e0e16701bee9ef8699604c966003879 Mon Sep 17 00:00:00 2001
From: Victor Chang <vichang@google.com>
Date: Mon, 4 May 2020 18:31:09 +0100
Subject: [PATCH 4/8] export libandroidicu headers instead of
 libandroidicu_headers

They are the same thing actually.

Bug: 155031873
Test: m droid
Change-Id: I577966a66ce780c8b6cff2d09333d1f030a16bfa
---
 core/fxcrt/Android.bp | 8 ++------
 1 file changed, 2 insertions(+), 6 deletions(-)

diff --git a/core/fxcrt/Android.bp b/core/fxcrt/Android.bp
index cfbe939eb..d48b2848c 100644
--- a/core/fxcrt/Android.bp
+++ b/core/fxcrt/Android.bp
@@ -4,12 +4,8 @@ cc_library_static {
 
     visibility: ["//external/pdfium:__subpackages__"],
 
-    header_libs: [
-        "libandroidicu_headers",
-    ],
-
-    export_header_lib_headers: [
-        "libandroidicu_headers",
+    export_shared_lib_headers: [
+        "libandroidicu",
     ],
 
     static_libs: [
-- 
2.40.1

